Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -578,6 +578,27 @@ public let CodeGenerators: [CodeGenerator] = [
         }
     },
 
+    CodeGenerator("ForOfWithDestructLoopGenerator", input: .iterable) { b, obj in
+        // Don't run this generator in conservative mode, until we can track array element types
+        guard b.mode != .conservative else { return }
+        var indices: [Int] = []
+        for idx in 0..<Int.random(in: 1..<5) {
+            withProbability(0.8) {
+                indices.append(idx)
+            }
+        }
+
+        if indices.count == 0 {
```

Comment:
Nit: can we instead just ensure that `indices.count > 0` and drop this case? How about just adding
```swift
if indices.isEmpty {
    indices = [0]
}
```

which has the same effect, but still generates a `ForOpWithDestruct` in this generator

Reply:
sounds good, I've added the the changes